platform-sdk/swirlds-common/src/testFixtures/java/com/swirlds/common/test/fixtures/merkle/util/LaggingLearningSynchronizer.java

```diff
@@ -53,9 +49,9 @@ public LaggingLearningSynchronizer(
     /**
      * {@inheritDoc}
      */
-    @Override
-    protected AsyncOutputStream<QueryResponse> buildOutputStream(
-            final StandardWorkGroup workGroup, final SerializableDataOutputStream out) {
-        return new LaggingAsyncOutputStream<>(out, workGroup, latencyMilliseconds, reconnectConfig);
-    }
+    //    @Override
+    //    protected AsyncOutputStream<QueryResponse> buildOutputStream(
+    //            final StandardWorkGroup workGroup, final SerializableDataOutputStream out) {
+    //        return new LaggingAsyncOutputStream<>(out, workGroup, latencyMilliseconds, reconnectConfig);
+    //    }
```

Comment:
Please nuke the commented-out code. Git will remember this if you ever need to restore the method.

Reply:
This code still makes sense, I just don't know how to implement it in the new model. I have it in my TODO list. At this point I don't have a goal to have 100% clear code

Reply:
It would be best not to have commented out code though, but okay.